src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Queries/CreateVenue.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using System;
-using Dfc.CourseDirectory.Core.DataStore.CosmosDb;
+using Dfc.CourseDirectory.Core.Models;
 using OneOf.Types;
 
-namespace Dfc.CourseDirectory.Testing.DataStore.CosmosDb.Queries
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries
 {
     public class CreateVenue : ICosmosDbQuery<Success>
     {
         public Guid VenueId { get; set; }
         public int ProviderUkprn { get; set; }
-        public string VenueName { get; set; }
+        public string Name { get; set; }
         public string Email { get; set; }
-        public string Telephone { get; set; }
+        public string PhoneNumber { get; set; }
```

Comment:
This is `Telephone` in the storage model. Surely we should maintain consistency across the domain with the storage model taking precedence?

Reply:
This was changed to align with the other models (like in the Update flow). The current model is a total mess so I sided with the VM in this case.

Reply:
I think the VM was the wrong choice. We should be aligning to our storage model convention, specifically the SQL storage model convention as that will be the source of truth moving forward. We should also be aiming for consistency across all models (except in the case of the Cosmos storage model as that's impractical to change)

Reply:
I'll change to `Telephone` for this Create query.